src/app/debug/table/table-settings-modal/table-settings-modal.component.ts

```diff
@@ -105,7 +105,7 @@ export class TableSettingsModalComponent implements OnDestroy {
     localStorage.setItem('transformationEnabled', form.transformationEnabled.toString());
     this.httpService.postTransformation(form.transformation).subscribe();
     const generatorEnabled: string = String(form.generatorEnabled === 'Enabled');
-    let data: any = {
+    let data: Object = {
```

Comment:
This is technically correct, but wouldnt it be nice to know what's in the object type wise?
This goes for everything that uses the `Object` type in this pr

Reply:
i'll work on fixing that

Reply:
i changed all the Objects that i could find. They should be more descriptive than just "object" now. If i missed any, just let me know.